client/app/email-domain/general-informations/email-domain-general-informations.html

```diff
@@ -1,125 +1,105 @@
-                                                  data-ng-if="ctrlTabGeneralInformations.domain.offer !== 'redirect'"
-                                                  data-uib-tooltip="{{ 'email_tab_accounts_quota_info' | translate }}"
-                                                  data-tooltip-placement="right"></span>
-                                        </strong>
-                                        <oui-spinner data-size="s" data-ng-if="ctrlTabGeneralInformations.loading.quotas"></oui-spinner>
-                                        <span data-ng-bind="(ctrlTabGeneralInformations.summary.account || '0') + ' / ' + (ctrlTabGeneralInformations.quotas.account || '0')"
-                                              data-ng-if="!ctrlTabGeneralInformations.loading.quotas"></span>
-                                    </div>
-                                </li>
-                                <li class="oui-tile__item">
-                                    <div class="oui-tile__definition">
-                                        <strong class="oui-tile__term d-block"
-                                                data-translate="email_tab_responders_quota"></strong>
-                                        <oui-spinner data-size="s" data-ng-if="ctrlTabGeneralInformations.loading.quotas"></oui-spinner>
-                                        <span data-ng-bind="(ctrlTabGeneralInformations.summary.responder || '0') + ' / ' + (ctrlTabGeneralInformations.quotas.responder || '0')"
-                                              data-ng-if="!ctrlTabGeneralInformations.loading.quotas"></span>
-                                    </div>
-                                </li>
-                                <li class="oui-tile__item">
-                                    <div class="oui-tile__definition">
-                                        <strong class="oui-tile__term d-block"
-                                                data-translate="email_tab_redirections_quota"></strong>
-                                        <oui-spinner data-size="s" data-ng-if="ctrlTabGeneralInformations.loading.quotas"></oui-spinner>
-                                        <span data-ng-bind="(ctrlTabGeneralInformations.summary.redirection || '0') + ' / ' + ((ctrlTabGeneralInformations.quotas.redirection + ctrlTabGeneralInformations.quotas.alias) || '0')"
-                                              data-ng-if="!ctrlTabGeneralInformations.loading.quotas"></span>
-                                    </div>
-                                </li>
-                            </ul>
+                        <div>
+                            <span data-ng-bind="ctrlTabGeneralInformations.serviceInfos.contactTech"></span>
+                            <small data-translate="global_technical"></small>
                         </div>
-                    </div>
-                </div>
-            </div>
+                        <div>
```

Comment:
Looks more like a `p`, maybe you can change `global_billing ` to have everything in a `p`